scripts/llms_config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "projectName": "kluster",
+  "projectUrl": "https://kluster.ai/",
+  "projectDescription": "kluster is the developer AI cloud platform to deploy, scale, and fine-tune models at lightning speed.",
+  "sectionPriority": ["get-started", "tutorials", "api-reference"],
+  "categories": ["Models", "OpenAI Compatibility", "Real-time Inference", "Batch Inference", "Fine-tuning", "Verify and Reliability", "Integrations", "MCP", "Dedicated Deployments", "Platform Management", "Tutorials", "Use Cases", "Best Practices"],
```

Comment:
maybe just need to update the section priority to match new top-level section (verify) and remove the categories for now